lib/Location.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Estáis usando mayúsculas en uno, minúsculas en otros ficheros. ¿Cuál es la convención?

Reply:
Nuestra convención es nombre de ficheros en minúscula con kebab case, lo cambiamos el otro día, pero parece que no ha surtido efecto. Lo rectificamos